yarn.lock

```diff
@@ -15371,28 +15851,31 @@ leaflet@0.7.x:
   resolved "https://registry.yarnpkg.com/leaflet/-/leaflet-0.7.7.tgz#1e352ba54e63d076451fa363c900890cb2cf75ee"
   integrity sha1-HjUrpU5j0HZFH6NjyQCJDLLPde4=
 
-lerna@^3.18.1:
-  version "3.19.0"
-  resolved "https://registry.yarnpkg.com/lerna/-/lerna-3.19.0.tgz#6d53b613eca7da426ab1e97c01ce6fb39754da6c"
-  integrity sha512-YtMmwEqzWHQCh7Ynk7BvjrZri3EkSeVqTAcwZIqWlv9V/dCfvFPyRqp+2NIjPB5nj1FWXLRH6F05VT/qvzuuOA==
-  dependencies:
-    "@lerna/add" "3.19.0"
-    "@lerna/bootstrap" "3.18.5"
-    "@lerna/changed" "3.18.5"
-    "@lerna/clean" "3.18.5"
-    "@lerna/cli" "3.18.5"
-    "@lerna/create" "3.18.5"
-    "@lerna/diff" "3.18.5"
-    "@lerna/exec" "3.18.5"
-    "@lerna/import" "3.18.5"
-    "@lerna/init" "3.18.5"
-    "@lerna/link" "3.18.5"
-    "@lerna/list" "3.18.5"
-    "@lerna/publish" "3.18.5"
-    "@lerna/run" "3.18.5"
-    "@lerna/version" "3.18.5"
-    import-local "^2.0.0"
-    npmlog "^4.1.2"
+lerna@^5.5.0:
+  version "5.5.0"
```

Comment:
Could a `yarn install` be run again, to bump lerna to `5.5.4` please? Regarding https://github.com/ovh/manager/security/dependabot/399.

More information: 

```
$ yarn why parse-url
[...]
=> Found "parse-url@6.0.2"
info Reasons this module exists
   - "_project_#git-url-parse#git-up" depends on it
   - Hoisted from "_project_#git-url-parse#git-up#parse-url"
[...]
```
`parse-url` is deep dependency of `git-url-parse`, then

```
$ yarn why git-url-parse
[...]
=> Found "git-url-parse@11.6.0"
info Has been hoisted to "git-url-parse"
info Reasons this module exists
   - "workspace-aggregator-72c7e44d-ea1c-4dc0-971c-94e680fcacb3" depends on it
   - Specified in "devDependencies"
   - Hoisted from "_project_#git-url-parse" (to remove in other PR)
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#github-client#git-url-parse" bring as lerna deep dependency
[...]
```